crates/driver/src/tests/setup/driver.rs

```diff
@@ -328,7 +328,7 @@ async fn create_config_file(
                 .map(|abs| abs.0)
                 .unwrap_or_default(),
             solver.slippage.relative,
-            const_hex::encode(solver.private_key.secret_bytes()),
+            solver.signer.to_bytes(),
```

Comment:
Is `to_bytes()` also encoding the `0x` prefix or is this a breaking change in the format?